src/Ledger/hs-src/test/UtxowSpec.hs

```diff
@@ -29,6 +29,11 @@ initParams = MkPParams
   , ppMaxHeaderSize = 100
   , ppMaxValSize = 1000
   , ppMinUTxOValue = 0
+  , ppMinFeeRefScriptCoinsPerByte = (1, 10)
```

Comment:
```suggestion
```
This is defined on line 45 as well. However, it's set to 0 there, so you might want to make it `(1, 10)` on line 45.